README.rst

```diff
@@ -72,23 +83,22 @@ the latest release with:
     pip install pip -U
     pip install -e .
 
+
 Dependencies
 ------------
 The ``pytwin`` package requires access to an Ansys License Server
-with the ``twin_builder_deployer`` feature available (see the
-`Getting started`_ section).
+with the ``twin_builder_deployer`` feature available. For more information,
+see `Getting started`_.
 
 
 License and acknowledgments
 ---------------------------
 PyTwin is licensed under the MIT license.
 
-For more information on `Ansys Digital Twins`_, see the `Twin Builder`_
-page on the Ansys website.
 
 .. LINKS AND REFERENCES
-.. _Ansys Digital Twins: https://www.ansys.com/products/digital-twin/
-.. _Twin Builder: https://www.ansys.com/products/digital-twin/ansys-twin-builder
+.. _Digital Twins: https://www.ansys.com/products/digital-twin/
+.. _Ansys Twin Builder: https://www.ansys.com/products/digital-twin/ansys-twin-builder
 .. _Documentation: https://twin.docs.pyansys.com/
 .. _Getting started: https://twin.docs.pyansys.com/release/0.1/getting_started/index.html
```

Comment:
```suggestion
.. _Getting started: https://twin.docs.pyansys.com/release/0.2/getting_started/index.html
```
Change the version pointing to latest. This is a bug and we will try to keep the latest stable in the next release of pyansys/actions. Till then, you might have to change it manually.

Reply:
Thanks @Revathyvenugopal162!